modules/tools/sonarqube/variables.tf

```diff
@@ -1,6 +1,29 @@
-variable "enable_sonarqube" {
-  default = true
+variable "admin_password" {
+  type      = string
+  sensitive = true
+}
+
+variable "force_authentication" {
+  default     = true
+  description = "Require authentication. Set to false to enable anoymouse access."
```

Comment:
```suggestion
  description = "Require authentication. Set to false to enable anonymous access."
```

Reply:
also, this variable doesn't appear to be used.  the chart values hardcode `sonar.forceAuthentication` to `true`